src/orga/subteams.rst

```diff
@@ -2,6 +2,66 @@ A list of current sub-teams
 ***************************
 
 
+Code of Conduct Sub-Team
+=============================
+
+Role
+----
+The code-of-conduct (CoC) sub-team is responsible for accepting, handling
+and resolving code-of-conduct related items (including but not limited to
+CoC violations, enforcement, communications, warnings, etc.) on conda-forge's
+communication channels (including but not limited to Gitter, GitHub, Keybase, the various
+mailing lists, etc) pursuant to conda-forge's code of conduct document(s).
+
+Charter
+-------
+Static
+
+Responsibility
+--------------
+The CoC sub-team will operate via the following rules:
+
+- It can only be comprised of ``core`` members.
```

Comment:
The conda-incubator CoC team does not have an equivalent clause and, more specifically, they explicitly mention an external consultant when difficult situations arise. See https://github.com/conda-incubator/governance/blob/main/CODE_OF_CONDUCT.md#ombudsperson.

It might be useful for us too.